clang/runtime/dpct-rt/include/util.hpp.inc

```diff
@@ -283,6 +283,28 @@ vectorized_isgreater<sycl::ushort2, unsigned>(unsigned a, unsigned b) {
 }
 // DPCT_LABEL_END
 
+// DPCT_LABEL_BEGIN|vectorized_isgreater_uchar4|dpct
+// DPCT_DEPENDENCY_EMPTY
+// DPCT_CODE
+/// Compute vectorized isgreater for two unsigned int values, with each value
+/// treated as a vector of four unsigned char.
+/// \param [in] a The first value
+/// \param [in] b The second value
+/// \returns The vectorized greater than of the two values
+template<>
+inline unsigned
+vectorized_isgreater<sycl::uchar4, unsigned>(unsigned a, unsigned b) {
+  sycl::vec<unsigned, 1> v0{a}, v1{b};
+  auto v2 = v0.template as<sycl::uchar4>();
+  auto v3 = v1.template as<sycl::uchar4>();
+  sycl::uchar4 v4;
+  v4[0] = v2[0] > v3[0];
+  v4[1] = v2[1] > v3[1];
```

Comment:
here also need cal the 2nd & 3rd channel.  the result should be: v4[0] = v2[0] > v3[0]? 0xff:0;